src/main/resources/org/kuali/kfs/kim/spring-kim.xml

```diff
@@ -0,0 +1,415 @@
+<!--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2021 Kuali, Inc.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<!--Cornell Customization: backport redis-->
```

Comment:
I thought this would make it easier to spot that this was backported for redis and could be easily removed altogether when we upgrade to the 10/21/21 version of financials but I can add the changes to the cu-spring-kim.xml if you think that would be cleaner.

Reply:
Our other backports of Spring bean changes have typically placed the beans in our CU-specific Spring files. I think it would make sense to do that here as well, especially since there were only a couple beans that were changed for the redis work.

Reply:
changed as requested